service-runner/src/main/java/org/stellar/anchor/platform/ServiceRunner.java

```diff
@@ -60,7 +61,17 @@ static ConfigurableApplicationContext startSepServer() {
   }
 
   static void startStellarObserver() {
-    StellarObservingService.start();
+    String strPort = System.getProperty("OBSERVER_HEALTH_SERVER_PORT");
+    int port = DEFAULT_OBSERVER_HEALTH_SERVER_PORT;
+    if (strPort != null) {
+      port = Integer.parseInt(strPort);
+    }
+    String contextPath = System.getProperty("SEP_CONTEXTPATH");
+    if (contextPath == null) {
+      contextPath = DEFAULT_CONTEXTPATH;
```

Comment:
💡 `STELLAR_OBSERVER_DEFAULT_CONTEXTPATH`?

Reply:
Addressed in https://github.com/stellar/java-stellar-anchor-sdk/pull/602/commits/1a530f5a0e2688076187295c551b7d59d4458d5a.